Source/NSPageController.m

```diff
@@ -0,0 +1,199 @@
+/* Implementation of class NSPageController
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: 27-07-2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import <Foundation/NSArray.h>
+#import <Foundation/NSKeyValueObserving.h>
+#import <Foundation/NSArchiver.h>
+
+#import "AppKit/NSPageController.h"
+#import "AppKit/NSView.h"
+
+@implementation NSPageController
+
+- (instancetype) init
+{
+  self = [super init];
+  if (self != nil)
+    {
+      _transitionStyle = NSPageControllerTransitionStyleStackHistory;
+      _delegate = nil;
+      _arrangedObjects = [[NSMutableArray alloc] initWithCapacity: 10];
```

Comment:
I am surprised that there is no NSArrayController involved in this class.

Reply:
> Merging while you see that I am in the middle of a review is a rather strange behaviour. At least I expect that you still address the open issues before moving on to another feature branch.

Sorry!  I thought you were done.  I will do so.